packages/orbit-components/src/Slider/components/Handle/index.tsx

```diff
@@ -87,8 +86,7 @@ const Handle = ({
         onTop && "z-20",
       )}
       style={{
-        // TODO use inset-inline-start once we don’t support ios_safari 14
-        [leftRight({ theme })]: `calc(${left}% - ${theme.orbit.space300})`,
+        insetInlineStart: `calc(${left}% - ${theme.orbit.space300})`,
```

Comment:
I'm not sure if my question is not out of the scope of this PR, but I noticed that `leftRight({ theme })` is also used in `Bar/index.tsx` file. I tried to replace it with `insetInlineStart` locally and it seems that it looks the same. Why was not changed this property in that file as well?

Reply:
Well spotted. Can be removed indeed. Thanks